external-crates/move/crates/move-package/src/lock_file/schema.rs

```diff
@@ -60,6 +60,16 @@ pub struct Dependency {
     pub digest: Option<String>,
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct CompilerToolchain {
+    /// The Move compiler version used to compile this package.
+    #[serde(rename = "compiler-version")]
+    pub compiler_version: String,
+    /// The Move compiler flags used to compile this package.
+    #[serde(rename = "compiler-flags")]
+    pub compiler_flags: Vec<String>,
```

Comment:
See PR description for more, but essentially: Do these schema changes cover what we need to store to support building compiler versions?

Reply:
I think so, yes.